t/e2e/whisper/whisper_jail_test.go

```diff
@@ -63,11 +63,13 @@ func (s *WhisperJailTestSuite) TestJailWhisper() {
 	s.StartTestBackend()
 	defer s.StopTestBackend()
 
+	r := s.Require()
+
 	keyPairID1, err := s.AddKeyPair(TestConfig.Account1.Address, TestConfig.Account1.Password)
-	s.NoError(err)
+	r.NoError(err)
```

Comment:
Would like to see a `s.Require().NoError()` here and below as above. It's more documenting.

Reply:
or rename `r` to `require`. should also be pretty descriptive.

Reply:
I can rename it to `require` but I was thinking about `r` as something as fundamental as `t *testing.T` :)

Reply:
Hmm, `t` is a language standard, `r` only by a chosen test suite. So at least `require`. 😉

Reply:
I would say `t` is a convention rather than a standard and `r` can also be treated as a convention in our repo. EOT, I will change it as it's used only here now :)

Reply:
Okay, wording hasn't been optimal, convention is better. 👍 

In my own `goaudit` I create an `assert` based on the passed `testing.T`. And then the asserts are `asssert.NotNil()`, `assert.NoError()`, `assert.Contains()`, etc.

Reply:
No need for change as it is our convention here.